dev/MRTCore/build/versions.props

```diff
@@ -24,6 +24,7 @@
     <DotNetCoreRuntimeVersion>5.0.8</DotNetCoreRuntimeVersion>
     <DetoursVersion>4.0.1</DetoursVersion>
     <CppWinRTVersion>2.0.210806.1</CppWinRTVersion>
+    <WILVersion>1.0.210803.1</WILVersion>
```

Comment:
><WILVersion>1.0.210803.1</WILVersion> [](http://example.com/codeflow?start=4&length=37)

Is this used?

Reply:
I'm not sure. I was looking for places where CppWinRTVersion was defined/used and saw this. I was hesitant to add this because it was outside of the dev/VSIX directory, but figured it'd be better to add then remove if unnecessary rather than omit and forget.

With that in mind, if this doesn't contribute to making the WIL nuget available to the C++ project templates, we should remove it. Do you agree?

Reply:
> With that in mind, if this doesn't contribute to making the WIL nuget available to the C++ project templates, we should remove it. Do you agree?

This is in `dev/MRTCore/build`. What's that got to do with the templates? Aren't they under `dev/VSIX`?

Reply:
This was a mistake and is removed in the most recent push.